libraries/AP_HAL_Linux/RCInput_RCProtocol.cpp

```diff
@@ -35,7 +35,7 @@ using namespace Linux;
 /*
   open a SBUS UART
  */
-int RCInput_RCProtocol::open_sbus(const char *path)
+int RCInput_RCProtocol::open_com(const char *path, const bool is_sbus)
```

Comment:
I would prefer not to have a `is_sbus` parameter. Please pass the options like baudrate and stopbits rather than a is_sbus

Reply:
I appreciate your point, however, I don't have the time to do an exhausive comparison between the serial setups on these interfaces. There are more differences than just the stopbit and buadrate, but those the one I recognized. This fix is not a perfect generalization, but it is an obvious improvement to what is currently there. SRXL is currently broken...